app/services/PsrSubmissionService.scala

```diff
@@ -60,11 +62,13 @@ class PsrSubmissionService @Inject()(
           minimalRequiredSubmission = minimalRequiredSubmission,
           checkReturnDates = checkReturnDates,
           loans = Option.when(schemeHadLoans)(Loans(schemeHadLoans, loanTransactionsTransformer.transformToEtmp(srn))),
-          assets = Option.when(landOrPropertyHeld || moneyWasBorrowed)(
+          assets = Option.when(landOrPropertyHeld || moneyWasBorrowed || disposeAnyLandOrProperty)(
```

Comment:
What happens in a scenario when there are properties (or borrowings) but no disposals?

Reply:
```
 optDisposedPropertyTransaction =
                      Option.when(disposeAnyLandOrProperty)(buildOptDisposedPropertyTransactions(srn, index))
```

Then optDisposedPropertyTransaction will be None and we'll send nothing to BE

Reply:
Is that right though? We should send landOrProperty (and borrowing) even if there are no disposals.

Reply:
yes I mean, we are not going to send any DisposedPropertyTransaction (if false) but landOrProperty (and borrowing).